### PR TITLE
Mobile authentication

### PR DESCRIPTION
**Pull request by ARtoriouSs**
- Add possibility to render json instead of redirect on auth for mobiles
- Remove all unused queries and tests for them
- Add `reload` helper for specs

### CODE REVIEW

File: lib/squadster_web/schema.ex [hunk 1]
```diff
@@ -8,47 +8,29 @@ defmodule Squadster.Schema do
...
-
-      resolve &AccountsResolver.user/3
-    end
-
     @desc "Get current user"
     field :current_user, :user do
       resolve &AccountsResolver.current_user/3
     end
 
+    # TODO: rebuild permissions
```
Comment (ARtoriouSs) [inline lib/squadster_web/schema.ex:16]: It'll be a separate PR

**Review by mirterious — APPROVED** [review]
